pilot/docker/prepare_proxy.sh

```diff
@@ -1,23 +1,26 @@
 #!/bin/bash
 # Envoy initialization script responsible for setting up port forwarding.
 
-set -o errexit
-set -o nounset
-set -o pipefail
+set -x # echo on
 
 usage() {
-  echo "${0} -p PORT -u UID [-h]"
+  echo "${0} -p PORT -u UID [-i CIDR] [-x CIDR] [-n] [-h]"
```

Comment:
In general, I would prefer environment variables - for consistency with 'raw vm' and because we may pass extra env via inject (using annotations or flags).

Reply:
done